commit db55ef8073facd9622cbc6ac1be52abeca269126
Author: Jon Bernard <jbernard@tuxion.com>
Date:   Mon May 13 11:38:17 2013 -0400

    Remove bashism in lttng-syscalls-generate-headers.sh
    
    Options to echo are not portable. In particular, the 'echo -e' option is
    implemented by some shells, including bash, to expand escape sequences.
    However, dash is one of the other family of shells that instead expands
    escape sequences by default.
    
    The printf command is portable and much more reliable.
    
    Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>

diff --git a/instrumentation/syscalls/lttng-syscalls-generate-headers.sh b/instrumentation/syscalls/lttng-syscalls-generate-headers.sh
index 046dc06..1754ae8 100755
--- a/instrumentation/syscalls/lttng-syscalls-generate-headers.sh
+++ b/instrumentation/syscalls/lttng-syscalls-generate-headers.sh
@@ -59,7 +59,7 @@ if [ "$CLASS" = integers ]; then
 
 NRARGS=0
 
-echo -e \
+printf \
 'SC_DECLARE_EVENT_CLASS_NOARGS(syscalls_noargs,\n'\
 '	TP_STRUCT__entry(),\n'\
 '	TP_fast_assign(),\n'\
